test/route.go

```diff
@@ -20,18 +20,34 @@ package test
 import (
 	"encoding/json"
 
+	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
 	"go.uber.org/zap"
 )
 
-// CreateRoute creates a route in the given namespace using the route name in names
-func CreateRoute(logger *zap.SugaredLogger, clients *Clients, names ResourceNames) error {
-	route := Route(Flags.Namespace, names)
+func logRoute(logger *zap.SugaredLogger, route *v1alpha1.Route) {
 	// Log the route object
 	if routeJSON, err := json.Marshal(route); err != nil {
 		logger.Infof("Failed to create json from route object")
 	} else {
-		logger.Infow("Created resource object", "ROUTE", string(routeJSON))
+		logger.Infow("resource", "ROUTE", string(routeJSON))
```

Comment:
This should be a helper method, so it's easy to change the format if necessary.

Reply:
Updated